src/tests/backend/specs/enforceLowerCasePadIds.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+const assert = require('assert').strict;
+const common = require('../common');
+const padManager = require('../../../node/db/PadManager');
+const settings = require('../../../node/utils/Settings');
+
+describe(__filename, function () {
+  this.timeout(30000);
+  let agent;
+  const cleanUpPads = async () => {
+    const padIds = ['UPPERCASEpad', 'uppercasepad', 'ALREADYexistingPad', 'alreadyexistingpad'];
+    await Promise.all(padIds.map(async (padId) => {
+      if (await padManager.doesPadExist(padId)) {
+        const pad = await padManager.getPad(padId);
+        await pad.remove();
+      }
+    }));
+  };
+
+  before(async function () { agent = await common.init(); });
+  beforeEach(async function () {
+    await cleanUpPads();
+  });
+  afterEach(async function () {
+    await cleanUpPads();
+  });
+
+  describe('not activated', function () {
+    Object.assign(settings, {
+      enforceLowerCasePadIds: false,
+    });
```

Comment:
This must happen inside a `before` or `beforeEach` hook, and the original value should be restored in an `after` or `afterEach` hook:

```suggestion
    let backup;

    before(async () => {
      backup = settings.enforceLowerCasePadIds;
      settings.enforceLowerCasePadIds = false;
    });

    after(async () => {
      settings.enforceLowerCasePadIds = backup;
    });

```

Reply:
applied: e3a9eb75e7d4093ed0f8083201f2068f789f6b16